kuma/javascript/src/error-boundaries.jsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import { gettext } from './l10n.js';
+
+export class AppErrorBoundary extends React.Component {
+    state = { error: null };
+
+    static getDerivedStateFromError(error) {
+        return { error };
+    }
+
+    logError() {
+        // https://github.com/mozilla/kuma/issues/5833
+        // When we start to use Sentry, change the method signature to
+        // logError(error, errorInfo) {
+    }
+
+    componentDidCatch(error, errorInfo) {
+        document.title = 'Application rendering Error';
+        this.logError(error, errorInfo);
+    }
+
+    render() {
+        if (this.state.error) {
+            return (
+                <ErrorWrapper title={gettext('Application rendering error')}>
+                    <p>
```

Comment:
nit: Is there a reason for why we're supplying children to this component when they're the same in both instances?

Reply:
No reason. I think I had bigger ideas and intentions but I've run out if ideas :)
Also, using the same English sentence for application errors and content errors makes for *fewer* things that need to be translated. I'm going to move this to the `<ErrorWrapper/>`

By the way, what do you think about the name `ErrorWrapper`?